Docs/topics/vita.md

```diff
@@ -1,96 +1,69 @@
 
 <include from="Library.topic" element-id="supported-version-notice"/>
 
-Allefresher Vita is a CFW plugin which patches LBP Vita to custom servers. This guide aims to show how to set it up on a modded Vita.
-
-To download Allefresher, you must first visit the [Releases Page](https://github.com/LittleBigRefresh/Allefresher/releases/latest). 
-Once you've opened that in your browser, select `Allefresher_vita.suprx` in the files listed.
-
-![Allefresher Artifacts](allefresher-artifacts.png)
-
-Once you have downloaded the CFW plugin, keep it on hand for later.
+Allefresher is a set of CFW plugins which patches LBP to custom servers. 
+This guide aims to show how to set it up on a modded Vita using AutoPlugin2.
 
 ## Step 1: Jailbreak your device and install a CFW
 
 We can't help you with this directly, but there are various guides out there you can go out and find. We personally recommend the guide on [vita.hacks.guide](https://vita.hacks.guide/). 
 We recommend installing the HENkaku Ensō CFW, as that is what we officially support.
 
-After you have jailbroken your Vita, install [VitaShell](https://github.com/TheOfficialFloW/VitaShell/releases/latest).
+## Step 2: Install AutoPlugin2
 
-## Step 2: Start and connect to an FTP server on your Vita
+<note>If you already have AutoPlugin2 installed, you can skip right to <a anchor="step-3-install-allefresher">Step 3</a>.</note>
 
-In VitaShell, press "Start", and change the "SELECT button" option to FTP. Press start again and then press select. 
-This will tell you the IP you need to type into Windows Explorer's address bar. (e.g. `ftp://192.168.0.100:1337/`).
-After pressing enter, you should see the list of files on your Vita.
+After you have jailbroken your Vita, we need to install [AutoPlugin2](https://github.com/ONElua/AutoPlugin2/releases/latest). 
+We will be doing this using [VitaShell](https://github.com/TheOfficialFloW/VitaShell/releases/latest), which you can download through VitaDeploy.
 
-![vita-windows-ftp.png](vita-windows-ftp.png)
+Open VitaShell, and press <shortcut>← + O</shortcut>, this will open up the camera to scan the QR code below. 
+After scanning the QR code, it should ask, "Do you want to install this package" with a URL starting with `https`. 
 
-## Step 3: Install iTLS
-
-iTLS is a plugin which enables TLS 1.2 (a more modern encryption standard) on Vita CFW. This is required for Allefresher Vita to function.
-
-Download iTLS from [here](https://github.com/SKGleba/iTLS-Enso/releases/latest/) and transfer it to your Vita through VitaShell, then install it. 
-Alternatively, you can also download it using [VitaDeploy](https://vita.hacks.guide/installing-vitadeploy.html).
+<warning>
+If it does not say exactly "Do you want to install this package", 
+then try to scan again after lowering your display's brightness, 
+you can also try placing the QR code on a different display, such as your phone.
+</warning>
```

Comment:
This reads poorly but I don't know enough context to fix it.

Reply:
basically if the vita's camera is super overexposed, sometimes ittl claim to read the QR code, but instead of asking "Do you want to install this package", ittl ask "Do you want to open this URL", with a blank URL, which seems to be the "failed to read QR code" condition in vitashell for some reason

Reply:
In that case...
```suggestion
<warning>
If it asks if you want to open a URL instead of installing a package, your Vita likely misread the QR code.
Try to scan again after lowering your display's brightness. You can also try placing the QR code on a different display, such as your phone.
</warning>
```